hack/increment-chart-version.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+usage() {
+  cat <<EOF
+Usage: $(basename "$0") <bump_type>
+
+Bumps the Helm chart version in Chart.yaml by the specified type.
+
+Required argument:
+  bump_type     The type of version bump. Must be one of:
+                  - major
+                  - minor
+                  - patch
+
+Examples:
+  $(basename "$0") patch     # 1.2.3 -> 1.2.4
+  $(basename "$0") minor     # 1.2.3 -> 1.3.0
+  $(basename "$0") major     # 1.2.3 -> 2.0.0
+
+EOF
+  exit 1
+}
+
+BUMP_TYPE="$1"
+
+if [[ -z "${BUMP_TYPE}" ]]; then
+  echo -e "Error: no \$bump_type passed as \$1.\n"
+  usage
+elif [[ "${BUMP_TYPE}" != "major" ]] && [[ "${BUMP_TYPE}" != "minor" ]] && [[ "${BUMP_TYPE}" != "patch" ]]; then
+  echo -e "Error: \$bump_type \"${BUMP_TYPE}\" not recognized.\n"
+  usage
+fi
+
+# requires git, yq
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# Path to your Chart.yaml
+CHART_FILE="charts/llm-d-modelservice/Chart.yaml"
+CHART_PATH="${REPO_ROOT}/charts/llm-d-modelservice/Chart.yaml"
+
+current_version=$(git show origin/main:${CHART_FILE} | yq e '.version' -)
+
+echo "Current version: $current_version"
```

Comment:
Somehow it could not identify the current version for the Chart.yaml currently at v0.2.16 

```
$ make bump-chart-version-patch
...
==== Running bump chart version ====
hack/increment-chart-version.sh patch
Current version: 0.0.5
Version updated: 0.0.5 → 0.0.6
```